pyanaconda/users.py

```diff
@@ -168,6 +169,25 @@ def validatePassword(pw, user="root", settings=None):
 
     return (valid, strength, message)
 
+def check_name(in_name, ret_bool = False):
+    ign_dir = {}
+    for inc in os.listdir("/") + ["root", "home", "daemon"]:
+        ign_dir.update({inc: "Cannot set user name, reserved for system: "})
+    ign_dir.update({r"^[\d]": "User name cannot start with a digit: "})
+    ign_dir.update({r"[^A-Za-z0-9\s]": "User name cannot contain any non-ASCII character: "})
+
+    for inc in ign_dir:
+        tmp = regsearch(inc, in_name)
```

Comment:
`match`

- can collide with function, if it is imported
- this is a temporary, I don't think we need to store it. This variable is changed often.